modules/rte_mis_core/src/Plugin/Block/SiteMenuTextSectionBlock.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Drupal\rte_mis_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'Site Menu Text Section' block with image and text fields.
+ *
+ * @Block(
+ *   id = "site__menu_text_section_block",
+ *   admin_label = @Translation("Site Menu Text Header & Footer Block"),
+ * )
+ */
+class SiteMenuTextSectionBlock extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build() {
+    $config = $this->getConfiguration();
+    $build = [];
+
+    $currentYear = date('Y');
+    $text = isset($config['block_text']) && $config['block_text'] !== '' ? $config['block_text'] : "Copyright © $currentYear. Department of School and Mass Education, Government of India. All Rights Reserved.";
```

Comment:
Same here - Could you keep the static string wrapped inside t()?

Reply:
ok